tests/test_feature.py

```diff
@@ -286,6 +287,21 @@ def test_process_folder(tmpdir):
     assert all(y.index.levels[2] == index.levels[1])
     np.testing.assert_array_equal(y.values, y_expected)
 
+    # non-existing folder
+    with pytest.raises(FileNotFoundError):
+        feature.process_folder('bad-folder')
+
+    # empty folder
+    root = str(tmpdir.mkdir('empty'))
+    y = feature.process_folder(root)
+    pd.testing.assert_frame_equal(
+        y,
```

Comment:
We have a dataframe here:

```suggestion
    df = feature.process_folder(root)
    pd.testing.assert_frame_equal(
        df,
```